drivers/wifi/eswifi/eswifi_core.c

```diff
@@ -141,6 +141,12 @@ int eswifi_at_cmd_rsp(struct eswifi_dev *eswifi, char *cmd, char **rsp)
 		return -EIO;
 	}
 
+	if (len == CONFIG_WIFI_ESWIFI_MAX_DATA_SIZE) {
```

Comment:
Could the len be larger than `CONFIG_WIFI_ESWIFI_MAX_DATA_SIZE`, so should we have `>=` here.

Reply:
Probably wont, but it doesn't hurt to add this